src/components/forms/JurisdictionMetadata/tests/index.test.tsx

```diff
@@ -100,3 +105,58 @@ describe('components/forms/JurisdictionMetadata', () => {
     });
   });
 });
+
+describe('uploads form', () => {
```

Comment:
Sorry if I missed it, but does this test assert that the metadata values (the actual data payload) is as we expect it?

Reply:
yes